app/presenters/result_set_presenter.rb

```diff
@@ -22,25 +23,52 @@ def initialize(finder, filter_params, view_context, sort_presenter, show_top_res
 
   def to_hash
     {
```

Comment:
This gets called a fair bit. Shall we wrap this in a `@to_hash ||= begin ... end`?

Reply:
I'm not familiar with that syntax. Can you be more specific?

Reply:
This result of this change is that the first time `to_hash` is called, we cache the result as a variable, and then rather than performing expensive operations again, we return the cached value.

```
def to_hash
   @to_hash ||= begin
      { data: big_request_that_takes_a_long_time }
   end
end
```